src/style.css

```
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.icon-search {
+  position: absolute;
+}
+
+.big-icon {
+  padding-left: 26px;
+  padding-top: 25px;
+  width: 19px;
+  height: 19px;
+}
+
+.small-icon {
+  padding-left: 13px;
+  padding-top: 15px;
+  width: 11px;
+  height: 11px;
+}
```

there is no need to keep both small and big icon classes. Please, check example from checklist and apply the same to search-bar. Just create search-bar class and its modifier  search-bar--small
```suggestion
.icon-search {
  position: absolute;
   padding-left: 26px;
  padding-top: 25px;
  width: 19px;
  height: 19px;
}

.small-icon {
  padding-left: 13px;
  padding-top: 15px;
  width: 11px;
  height: 11px;
}
```